concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetAccountInfoTest.java

```diff
@@ -0,0 +1,474 @@
+    private static final byte[] ENCRYPTED_AMOUNT = new byte[]{0, 0, 1};
+    private static final long ACCOUNT_AMOUNT = 10000;
+    private static final long ACCOUNT_INDEX = 1;
+    private static final byte[] ENCRYPTION_KEY = new byte[]{0, 0, 2};
+    private static final long RELEASE_AMOUNT = 10;
+    private static final long RELEASE_TIME = 1;
+    private static final byte[] RELEASE_TRANSACTION_HASH = new byte[]{0, 0, 3};
+    private static final long SEQUENCE_NUMBER = 2;
+    private static final long BAKER_ID = 1;
+    private static final byte[] BAKER_AGGREGATION_KEY
+            = ED25519PublicKey.from(ED25519SecretKey.createNew()).getBytes();
+    private static final byte[] BAKER_ELECTION_VERIFY_KEY
+            = ED25519PublicKey.from(ED25519SecretKey.createNew()).getBytes();
+    private static final byte[] BAKER_SIGNATURE_VERIFY_KEY
+            = ED25519PublicKey.from(ED25519SecretKey.createNew()).getBytes();
+    private static final String BAKER_POOL_URL = "www.example-baker-pool.com";
+    private static final int COMMISSION_BAKING_PPHT = 10;
+    private static final int COMMISSION_FINALIZATION_PPHT = 11;
+    private static final int COMMISSION_TRANSACTION_PPHT = 12;
+    private static final boolean RESTAKE_EARNINGS = true;
+    private static final long BAKER_REDUCE_STAKE_AMOUNT = 100;
+    private static final long BAKER_REDUCE_STAKE_TIME = 2;
+    private static final long STAKED_AMOUNT = 1000;
+    private static final int ACCOUNT_THRESHOLD = 10;
+    private static final int AR_THRESHOLD = 5;
+    private static final byte[] COMMITMENT_CRED_COUNTER = new byte[]{0, 0, 9};
+    private static final byte[] CREDENTIAL_REG_ID = com.concordium.sdk.transactions.CredentialRegistrationId
+            .from("8e5c75fda3f791efd025e7e8fb0c26c3e111211e375fc9423b1ca308a696140e44c90dcabe75108b41db625152acae50")
+            .getRegId();
+    private static final int IDP_ID = 999;
+    private static final int SIGNATURE_THRESHOLD = 8;
+    private static final byte[] NORMAL_CRED_SIG_0 = ED25519PublicKey.from(ED25519SecretKey.createNew()).getBytes();
+    private static final byte[] COMMITMENT_MAX_ACCOUNTS = new byte[]{10, 10, 10};
+    private static final byte[] COMMITMENT_PRF = new byte[]{11, 11, 11};
+    private static final byte[] COMMITMENT_ID_CRED_SHARING_COEFF = new byte[]{12, 12, 12};
+    private static final byte[] COMMITMENT_ATTRIBUTE_0 = new byte[]{13, 13, 13};
+    private static final YearMonth POLICY_CREATED_AT
+            = YearMonth.newBuilder().setYear(2022).setMonth(10).build();
+    private static final YearMonth POLICY_VALID_TO
+            = YearMonth.newBuilder().setYear(2024).setMonth(10).build();
+    private static final byte[] FIRST_NAME_VALUE
+            = "policy-value-0".getBytes(StandardCharsets.UTF_8);
+    private static final byte[] ENC_ID_PUB_SHARE = new byte[]{90, 90, 90};
+    private static final NormalCredentialValues.Builder NORMAL_CREDENTIAL_BUILDER = NormalCredentialValues.newBuilder()
```

Comment:
Why return the builder? You never extend it?

Reply:
This is the mocked return value? Which you are checking against the "expected" result declared below?

Reply:
Maybe name it NORMAL_CREDENTIAL_RESULT or something similar.

Reply:
removed it and merged with the Account Info Creation since there was only a single usage of the value.